simphony_ui/ui.py

```diff
@@ -0,0 +1,199 @@
+from traits.api import (HasStrictTraits, Int, Float, Enum, Instance,
+                        Str, Directory, BaseInt, BaseFloat, Array, File)
+from traitsui.api import View, Item, UItem, Tabbed, VGroup, HGroup
+import os
+
+
+class PositiveFloat(BaseFloat):
```

Comment:
put the new traits in a separate local_traits.py (_not_ traits.py, too much chance of conflicts with traits itself)